metadata.json

```diff
@@ -23,5 +23,10 @@
       "operatingsystemrelease": ["7", "8"]
     }
   ],
-  "data_provider": null
+  "requirements": [
+    {
+      "name": "puppet",
+      "version_requirement": ">= 4.0.0 < 5.0.0"
```

Comment:
Yes. Puppet 5 has not been released yet so we can't guarantee module compatibility with it.

Most Puppet modules on the forge pin like this, for example: https://forge.puppet.com/crayfishx/firewalld